### PR TITLE
[FABCI-482] Add Nightly Build

### PR DESCRIPTION
**Pull request by lindluni**
Signed-off-by: Brett Logan <brett.t.logan@ibm.com>

### CODE REVIEW

**Comment by lindluni** [comment]
Successful runs here: https://dev.azure.com/lindluni/Fabric/_build?definitionId=1&_a=summary

File: ci/azure-pipeline-nightly.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: $(SourceBranchName)-$(Date:yyyyMMdd)$(Rev:.rrr)
+
+trigger: none
+pr: none
+schedules:
+  - cron: "0 6 * * *"
+    displayName: Daily midnight build
```
Comment (rameshthoomu) [inline ci/azure-pipeline-nightly.yml]: Specify at what time (UTC/EST) these builds will trigger. helps for quick check. Seems this triggers at morning 6:00 AM EST.

**Review by rameshthoomu — COMMENTED** [review]
@btl5037 It seems we are not capturing the docker container logs in this PR. Any plans to pipe them out in the near future?

File: ci/azure-pipeline-nightly.yml (hunk 1 above)
Comment (lindluni) [reply ci/azure-pipeline-nightly.yml]: This triggers at 12:00AM EST. I will adjust it to an hour before daily PTE runs

Comment (lindluni) [reply ci/azure-pipeline-nightly.yml]: And add a comment as to what time it triggers

**Comment by lindluni** [comment]
I didn't have any intent to capture the docker logs, but we could add it as a PR on top of this one.

**Comment by rameshthoomu** [comment]
LGTM

**Review by rameshthoomu — APPROVED** [review]

**Review by suryalnvs — APPROVED** [review]

**Comment by lindluni** [comment]
Successful run with newest changes: https://dev.azure.com/lindluni/Fabric/_build/results?buildId=333&view=logs&j=a056a568-22bd-59c0-0684-a00a715aafad&t=5bb4130b-3721-5819-126f-95ecd04a23fa

**Review by scottz64 — APPROVED** [review]

**Pull request merged**